src/LogFileReader.cpp

```diff
@@ -75,11 +80,25 @@ bool LogFileReader::readAll(const QString& fileName)
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
     // TODO implement this first
-
+    QStringList lines = line.split(',');
```

Comment:
`,` is what we can call a delimiter, and we can put it in the namespace as a const QString.